### PR TITLE
#129 Tag import start notification

### PR DESCRIPTION
**Pull request by toniedzwiedz**
Added a little popup that tells the user that they can just relax and wait for the result.

### CODE REVIEW

File: content/src/main/content/jcr_root/apps/acs-tools/components/tag-maker/clientlibs/js/app.js [hunk 1]
```diff
@@ -68,6 +68,9 @@ angular.module('acs-tools-tag-maker-app', ['ngFileUpload', 'acsCoral', 'ACS.Tool
                 tagIds: []
             };
 
+            NotificationsService.add('info',
```
Comment (davidjgonzalez) [inline content/src/main/content/jcr_root/apps/acs-tools/components/tag-maker/clientlibs/js/app.js]: Can we use `NotificationsService.running(true);`  and then in `success(...) and or error(..)` and `NotificationsService.running(false);` 

https://github.com/Adobe-Consulting-Services/acs-aem-tools/blob/master/content/src/main/content/jcr_root/apps/acs-tools/components/base-page/clientlibs/js/angularjs/notifications.js

Comment (toniedzwiedz) [inline content/src/main/content/jcr_root/apps/acs-tools/components/tag-maker/clientlibs/js/app.js]: @davidjgonzalez  I didn't notice that utility. Thanks for the suggestion. I'll change the code, test it and update the PR.

**Comment by toniedzwiedz** [comment]
Added the calls to `NotificationService#running`
I shortened the `info` message but left it there as I personally found the words _the process_ (rendered when setting the `running` property to `true`) a little lacking in way of description.

**Comment by davidjgonzalez** [comment]
Thanks for the enhnancement!!

**Pull request merged**